src/tests/test_config.py

```diff
@@ -104,3 +115,31 @@ def test_defaults(self):
         self.assertEqual(
             self.config.safeGetInt(
                 'bitmessagesettings', 'maxaddrperstreamsend'), 500)
+
+    def test_disable_identity(self):
+        """Test Disable the identity"""
+        test_config_object = StringIO(test_config)
```

Comment:
maybe write a decorator (or maybe a new class) to do the init, and then write 6 methods

```
def setUp():
    self.enabled = Config()
    self.enabled.load(enabled_constant)
    self.disabled = Config()
    self.disabled.load(disabled_constant)
    self.nonexistent = Config()
```

Reply:
individual unit tests would be something like this:

```
self.enabled.enable(aldksfjal)
assertEquals(self.enabled.get(blabla), True)
```
```
assertRaises(self.nonexistent.enable(blablbla))
```